.github/workflows/release.yml

```diff
@@ -0,0 +1,116 @@
+name: release
```

Comment:
we can change it to every commit that is pushed to the main branch, but that will result in too many images (one for each commit). Is that acceptable?

Reply:
The dependencies dont change much. Also, we need to have an image whenever a dependency is updated, since that image needs to be used in the NDM repository.